src/main/java/com/offdk/play/model/game/MatchStatus.java

```diff
@@ -3,7 +3,7 @@
 public enum MatchStatus {
   CHALLENGED,
   ACCEPTED,
-  REJECTED,
+  REFUSED,
```

Comment:
To make things more consistent.  We had refuse and reject in interchangeably.  I am fine with either one as long as we use the same word.  The buttons currently have refuse.